astrology.lic

```diff
@@ -329,8 +329,10 @@ class Astrology
         check_weather
       when 'events'
         check_events(check_pools)
-      when 'moons'
-        check_moons
+      when 'moons'  #deprecated - left for backwards compat. only
+        check_attunement
+      when 'attunement'
```

Comment:
Youll want to change `astrology_training` in base.yaml to use `attunement` instead of `moons`
https://github.com/rpherbig/dr-scripts/blob/master/profiles/base.yaml#L757